src/index.ts

```diff
@@ -118,6 +118,42 @@ export const replacer = function replacer(options: Options) {
         keys.pop();
       }
 
+      if (typeof value === 'boolean') {
```

Comment:
not sure if it matters or not, but could extract the `typeof` into a constant and reuse that (and also use the `typeof` in the `undefined` case).

Should probably set up some benchmark suite in this repo so any optimizations can be verified